src/components/QuickOpenModal.js

```diff
@@ -302,6 +353,11 @@ export class QuickOpenModal extends Component<Props, State> {
       return null;
     }
 
+    let highlightedResults = results;
```

Comment:
I feel like we could do this differently considering the fact that we only need `highlightedResults` when there actually is something to highlight and we do not use `results` for the remainder of our story.